accessibility-oscilloscope/overlay/etc/init.d/S16oscope

```diff
@@ -21,7 +21,15 @@ enable-haptic-gpio
 
 # start programs in background
 
-pipe-wrapper "signal-compiler /dev/stdin /dev/stdout" /run/sigcomp-in /run/pnt-lut-pgm &
+# "left side" of tree
+
+pipe-wrapper "trigger-data -i /dev/stdin -o /dev/stdout" $OSCOPE_FIFO $SIGPROC_FIFO &
```

Comment:
does the `trigger-data` program terminate? if not, can just do `trigger-data -i $OSCOPE_FIFO -o $SIGPROC_FIFO`.

Reply:
yeah I modified it based on the feedback